megalinter/descriptors/robotframework.megalinter-descriptor.yml

```diff
@@ -0,0 +1,57 @@
+descriptor_id: ROBOTFRAMEWORK
+descriptor_type: tooling_format
+descriptor_flavors:
+  - cupcake
+  - formatters
```

Comment:
When having only one linter, it makes sense, but should this be on a linter level instead (if it's possible)?